static/compiler.js

```diff
@@ -117,6 +119,39 @@ define(function (require) {
             }
         });
 
+        this.outputEditor.addAction({
+            id: 'viewasmdoc',
+            label: 'View asm doc',
+            keybindings: [monaco.KeyMod.CtrlCmd | monaco.KeyCode.F8],
+            keybindingContext: null,
+            contextMenuGroupId: 'help',
+            contextMenuOrder: 1.5,
+            run: function (ed) {
```

Comment:
Just as a style point, I'd usually prefer this larger routines to be defined as a member function. This is fine as-is, but I might move it to make the construction routine a little shorter.